src/deploy/get.rs

```diff
@@ -41,6 +42,35 @@ mod deploy_hash {
     }
 }
 
+/// Handles providing the arg for the retrieval of the finalized approvals.
+mod finalized_approvals {
+    use super::*;
+
+    const ARG_NAME: &str = "get-finalized-approvals";
+    const ARG_SHORT: char = 'a';
+    const ARG_VALUE_NAME: &str = "BOOLEAN";
+    const ARG_HELP: &str =
+        "An optional flag specifying whether the finalized approvals are retrieved";
```

Comment:
I think this could be something like:

"If passed, the returned deploy approvals are the ones finalized in the block. Otherwise the approvals attached to the deploy when first received by the node will be returned"